apps/omg_watcher/lib/omg_watcher/api/transaction.ex

```diff
@@ -70,60 +40,4 @@ defmodule OMG.Watcher.API.Transaction do
     |> Transaction.Signed.encode()
     |> Client.submit(url)
   end
```

Comment:
This is really a good question for @pdobacz and @pthomalla !

Reply:
Huh? actually, I think neither of these 3 checks is implemented, or have I missed something major. Isn't it missing a remark, that it is the _intended_ behavior, but not yet implemented? @pnowosie 

If this is the case, I think the comment must be much more generic (intent only) and state clearly, what is not implemented yet (or maybe even don't say anything about security features at all, intent is expressed in the issues we're speculating about right now). Nice catch @jrhite!

Reply:
I think it's already mentioned that it's planned for not implemented yet?

> /transaction.submit is considered security critical because it checks validity of the chain before it forward the tx to the child chain. (Note: checking validity isn't yet implemented there - so it might confuse you).

Ref: https://github.com/omisego/elixir-omg/issues/1129#issuecomment-552859850

> It might have become clearer already, but to straighten out more: the truly "security related" bit of transaction.submit is not implemented, only planned for.


Ref: https://github.com/omisego/elixir-omg/issues/1129#issuecomment-553412020

Reply:
yes, but in this particular comment in code Jacob spotted, it sounds like it were implemented

Reply:
Aha I see. Will make the comment more obvious that it's planned but not implemented yet then! 🙏

Reply:
Updated in https://github.com/omisego/elixir-omg/pull/1123/commits/37768a17cddd4f442c0a3cf1a23ae09b23688aca